### ISSUE TITLE
Power Platform hub documentation linked files appear to be incomplete

### ISSUE THREAD

**Issue by lrwarren94** [MicrosoftDocs/power-platform#1664]
In the [Download the Assets](https://learn.microsoft.com/en-us/power-platform/guidance/adoption/wiki-community#download-the-assets) section of "Create an internal Microsoft Power Platform hub", [https://aka.ms/pphub-download](https://aka.ms/pphub-download) points to [PowerPlatformHub.zip](https://github.com/microsoft/coe-starter-kit/blob/main/CenterofExcellenceResources/Release/PowerPlatformHub/PowerPlatformHub.zip) within the [PowerPlatform Hub subfolder](https://github.com/microsoft/coe-starter-kit/tree/main/CenterofExcellenceResources/Release/PowerPlatformHub) of the coe-starter-kit repository.

We were running into issues with the Deploy-PowerPlatformHub.ps1 script from PowerPlatformHub.zip in our tenant (specifically, Invoke-PnPSiteTemplate fails every run), and as we were debugging we noticed a few things:

1. [The Source folder](https://github.com/microsoft/coe-starter-kit/tree/main/CenterofExcellenceResources/Release/PowerPlatformHub/Source) which contains images used by the template exists in the GitHub repo, but not PowerPlatformHub.zip. This leads to errors when Invoke-PnPSiteTemplate attempts to apply these images via the template. It is not clear in the documentation that one should download this Source folder from the GitHub repo in addition to PowerPlatformHub.zip.
2. Deploy-PowerPlatformHub.ps1 in the GitHub repo is a very different script from Deploy-PowerPlatformHub.ps1 in PowerPlatformHub.zip. We noticed that there are 2 Invoke-PnPSiteTemplate calls in the PowerPlatformHub.zip version and one in 1 call in the GitHub version.

I am working with our tenant admin to try the most recent GitHub repo version of the script, but we've still been running into errors (I'll open a separate issue for these if we don't solve them). 

Anyway, I'm just a little confused about the PowerPlatformHub.zip being misaligned with the GitHub version. I would understand if they were just different versions based on new updates, but the PowerPlatformHub.zip doesn't seem to work at all.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 9ff93b5f-b80f-45e4-7d90-9d22f5c56089
* Version Independent ID: 58c095c5-1963-bd82-cbca-a57ff90c91e0
* Content: [Create an internal Microsoft Power Platform hub - Power Platform](https://learn.microsoft.com/en-us/power-platform/guidance/adoption/wiki-community#test-if-the-deployment-was-successful)
* Content Source: [power-platform/guidance/adoption/wiki-community.md](https://github.com/MicrosoftDocs/power-platform/blob/main/power-platform/guidance/adoption/wiki-community.md)
* Service: **power-platform**
* Sub-service: **guidance**
* GitHub Login: @manuelap-msft
* Microsoft Alias: **mapichle**

**Comment by Laskewitz** [comment]
Hi @lrwarren94 -

Thanks for opening the issue.

I compared both files just now - and they are both the same files. Both the zip file and the powershell file have been updated last October 17th.

The image errors are a bit misleading - because the template engine checks first if the image already exists and when there's an error (that's what you'll see when you roll out this template for the first time), it will upload the image as a new file. That's why there's that many errors. In a future update we might add a toggle to show/hide the debugging output so that it's not worrying for people who deploy the template.

Did the site get deleted at the end of running the script? Or could you see the site after running the script and seeing the errors?

Kind regards,
Daniel

**Comment by lrwarren94** [comment]
Thanks @Laskewitz, looks like you're right about the files being the same. When we initially encountered errors with the original script, we modified it to add traceroutes and see what was going wrong. We must have accidentally modified or copy/pasted some of the original code in the process. Apologies for the confusion.

When we were running the original script, the site would be created, but the template would fail to apply. It would look like a normal community site with none of the custom content from the Power Platform Hub template. After the script failed, the site was not deleted--it remained and had to be manually deleted and cleared by our SharePoint admin. We repeated this while adding different traceroutes until we discovered errors like "The object id "site:812c10c9-b0ac-43d2-905c-569a1d39ed8f:web:4d2701dc-687c-4bd0-a349-a49a7cb58344" is invalid". We looked into the Source folder on the GitHub and noticed those GUIDs matched the image file names. Here are the traceroute logs (some information redacted):

- [2022-12-14.txt](https://github.com/MicrosoftDocs/power-platform/files/10249851/2022-12-14.txt)
- [20221214_1250.txt](https://github.com/MicrosoftDocs/power-platform/files/10249852/20221214_1250.txt)
- [20221214_1302.txt](https://github.com/MicrosoftDocs/power-platform/files/10249853/20221214_1302.txt)

I will go ahead and close this, though, because we eventually got the site to deploy correctly. What we had to do was revert to the original script file and run it after extracting both the contents of the template.pnp and the contents of the Source folder on GitHub to the root folder where Deploy-PowerPlatformHub.ps1 was. That gave us a clean run. After that, it had been deployed almost entirely correctly: the top bar menu navigation was missing, but we fixed that by digging through and finding a SharePoint setting that needed to be enabled. It all looks great now. 

Thanks for putting this out there for the community,
Lindsey
